dataverse/dataset.py

```diff
@@ -311,19 +319,19 @@ def upload_file(self, filename, content, zip_files=True):
         }
 
         requests.post(
-            self.edit_media_uri,
+            self.edit_media_uri.replace("https","http"),
```

Comment:
I'm confused. With this "replace" is the fix actually to switch from https to http? This reminds me of https://github.com/IQSS/dataverse/pull/5656 where we recently starting allowing http (not recommended) instead of https for password resets. We hope that this non-https configuration is only used for a test server!